app/lib/features/news/pages/add_news/add_news_page.dart

```diff
@@ -58,23 +58,18 @@ class AddNewsState extends ConsumerState<AddNewsPage> {
           nextSlide != null && nextSlide.type == UpdateSlideType.text;
       final changed = prevState?.currentUpdateSlide != nextSlide;
       if (isText && changed) {
-        final document = ActerDocumentHelpers.parse(
-          nextSlide.text ?? '',
-          htmlContent: nextSlide.html,
-        );
-
         final autoFocus =
             nextSlide.html?.isEmpty != false &&
             nextSlide.text?.isEmpty != false;
 
         setState(() {
           selectedNewsPost = nextSlide;
-          if (!document.isEmpty) {
-            // If the slide has content, update the editor state with it
-            textEditorState = EditorState(document: document);
-          } else {
-            // If no content, create a blank editor state
-            textEditorState = EditorState.blank();
+
+          if (isText && changed) {
```

Comment:
not 100% sure about this change, @kumarpalsinh25  might have some better idea, why we were checking for the document state before...